### PR TITLE
Dockerize

### PR DESCRIPTION
**Pull request by LittleExxGames**
Its a bit chaotic in what was added but search with ctrl+f for the docker compose and dockerfile for the dockerizing assignment.

Also, this is the same branch that works for the connecting the front end and back end. I fixed the display and now I can specify how much i want to show from the front end so you can check the /shoes endpoint. Lines 120-188 in index.js

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# syntax=docker/dockerfile:1
+
+FROM node:18-alpine3.19
+# WORKDIR /app
+COPY . .
```
Comment (bubbomb) [inline Dockerfile:5]: It looks like you're copying over every file including node_modules. You'll want to dockerignore the node_modules directory so you're not copying those over.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,201 @@
+
+
+const redis = require('redis');
+const host = 'localhost';
+const port = 6379;
```
Comment (bubbomb) [inline index.js]: These need to be reading from the environment variables

File: Dockerfile (hunk 1 above)
Comment (LittleExxGames) [reply Dockerfile:5]: and Done!

File: index.js (hunk 2 above)
Comment (LittleExxGames) [reply index.js]: Done!

**Pull request closed without merging**